### ISSUE TITLE
k4DataSvc seg faults if input file does not exist

### ISSUE THREAD

**Issue by tmadlener** [key4hep/k4FWCore#89]
When trying to open a non existing file, `k4run` will seg fault. This should be handled much more gracefully, as typos are otherwise pretty time consuming

**Comment by tmadlener** [comment]
This is potentially fixed with AIDASoft/podio#95. Need to check again

**Comment by jmcarcell** [comment]
This doesn't seem to happen anymore:
``` 
TFile::TFile        ERROR file /path/to/wrong_file.root does not exist
EventDataSvc        FATAL in sysInitialize(): standard std::exception is caught
```
Feel free to reopen if it seg faults again
